packages/hash/frontend/src/components/hooks/useCreatePage.ts

```diff
@@ -30,11 +34,13 @@ export const useCreatePage = (ownedById: string) => {
         variables: { ownedById, properties: { title: "", prevIndex } },
       });
 
-      const { ownedById: pageOwnedById, entityId: pageEntityId } =
-        response.data?.createPersistedPage ?? {};
+      const pageEntityId =
+        response.data?.createPersistedPage?.metadata.editionId.baseId;
 
-      if (pageOwnedById && pageEntityId) {
-        return router.push(`/${pageOwnedById}/${pageEntityId}`);
+      if (pageEntityId) {
+        const pageOwnedById = extractOwnedByIdFromEntityId(pageEntityId);
+        const pageEntityUuid = extractEntityUuidFromEntityId(pageEntityId);
```

Comment:
If [Ahmad's PR](https://github.com/hashintel/hash/pull/1479) merges first you could use his `splitEntityId` method here

Reply:
Yep I was going to actually go through and find all cases where we did this :)